### ISSUE TITLE
Remove Nigh-Invulnerability Belt and all vanilla items that can be abused!

### ISSUE THREAD

**Issue by boxitooo6629** [Atlantiss/NetherwingBugtracker#8115]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)
[//]: # (To find server revision type `.server info` command in the game chat.)

**Description**: 
https://www.wowhead.com/tbc/spell=30570/nigh-invulnerability-belt

**Current behaviour**:

**Expected behaviour**: 

**Server Revision**:

**Comment by ghost** [comment]
Im assuming you mean in arenas. Why disable the belt? So that your rogue/mage team can win every match within 30 seconds?

**Comment by boxitooo6629** [comment]
Yes, i mean in arenas. I don't play rouge mage in competative level so i have no idea what you are talking about. Reason I am saying it that pvp belt and any otherbelts will never be used since you have this. Killing the whole pvp and coming back to rouge mage this belt makes them even stronger. RM kills you with CC so if you are in CC you can't use the belt.

**Comment by ghost** [comment]
Here are some examples of why the belt should not be disabled in arena. (the guy is one of the best tbc priests) https://www.youtube.com/watch?v=i_B3D1OPEaQ

**Comment by boxitooo6629** [comment]
@369N Can you please look at some more reasonable video? https://youtu.be/FqtgkVrDATE?t=1143  

I selected the right section for you. Hope @Wolffenstein have a look too he was really silent about that topic.

**Comment by ghost** [comment]
"As someone who literally only plays double dps ... it's really frustrating when you're on a healer and then they belt at basically nothing and survive ..."

"I would rather the healer that we struggle so hard to kill sometimes with belt just dies faster"

Very enlightening video

**Comment by boxitooo6629** [comment]
What I am saying is this belt and net trinkets makes engi as an requirement in arena. If you are healer or rouge mage it's a must to have above 2.5k. Rouge mage can are win doesnt matter belt or not.

**Comment by tsu-1** [comment]
It shouldn't be removed.

This didn't shift the meta all around but actually gave some classes much-needed survivability.

As to, if this is a requirement or not; in every game, if you want to be on a top competitive level you will have to sacrifice and pick something that's meta - there is no way around it. Not having the engineering doesn't impact the game as much for other players that don't want to compete for Gladiator/Rank 1. They still will be able to win and gain rating.

**Comment by bluemanguy112** [comment]
Strongly disagree. It is no coincidence that basically all the high-rated RM are wearing the belt. While it surely helps, especially healers, from being rushed down too, it certainly also helps RM tremendously. 8sec of practically being immune to dmg while also not being affected by spellpushback (as mage) can help enormously as RM. Not to mention that for mage and rogue a backfire does not mean an instant-loss (as for most other classes) due to very potent deff cds (vanish/block).

If you aim to nerf Rogue/dps comps in general, the most efficient way is "Skull of Impending Doom", as it helps against the onslaught of CC and gives certain classes necessary uptime to win vs certain rogue comps. TBC classic has shown this.

Also it makes no sense to keep the item which practically adds another layer of RNG (backfire) onto an already RNG-heavy expansion while at the same time removing Skull which adds another layer of "skill" to the game.

TLDR: By keeping belt in the game the opposite of the desired effect is the case and comps like RM are getting buffed. Either remove all such items or enable skull.
